addons/mail/models/ir_ui_view.py

```diff
@@ -8,7 +8,7 @@ class View(models.Model):
     type = fields.Selection(selection_add=[('activity', 'Activity')])
 
     def _postprocess_tag_field(self, node, name_manager, node_info):
-        if node.xpath("ancestor::div[hasclass('oe_chatter')]"):
+        if node.xpath("ancestor::chatter"):
             # Pass the postprocessing of the mail thread fields
             # The web client makes it completely custom, and this is therefore pointless.
             name_manager.has_field(node, node.get('name'), {})
```

Comment:
To be verified, but all this override might no longer be necessary as there are no longer `<field` tags inside the chatter node.